editor/editor_node.cpp

```diff
@@ -6435,12 +6454,12 @@ EditorNode::EditorNode() {
 	p->add_shortcut(ED_SHORTCUT_AND_COMMAND("editor/quick_open_script", TTR("Quick Open Script..."), KeyModifierMask::CMD + KeyModifierMask::ALT + Key::O), FILE_QUICK_OPEN_SCRIPT);
 
 	p->add_separator();
-	PopupMenu *pm_export = memnew(PopupMenu);
-	pm_export->set_name("Export");
-	p->add_child(pm_export);
-	p->add_submenu_item(TTR("Convert To..."), "Export");
-	pm_export->add_shortcut(ED_SHORTCUT("editor/convert_to_MeshLibrary", TTR("MeshLibrary...")), FILE_EXPORT_MESH_LIBRARY);
-	pm_export->connect("id_pressed", callable_mp(this, &EditorNode::_menu_option));
+	export_as_menu = memnew(PopupMenu);
+	export_as_menu->set_name("Export");
+	p->add_child(export_as_menu);
+	p->add_submenu_item(TTR("Export As..."), "Export");
+	export_as_menu->add_shortcut(ED_SHORTCUT("editor/export_as_MeshLibrary", TTR("MeshLibrary...")), FILE_EXPORT_MESH_LIBRARY);
```

Comment:
It's not directly related to this PR but since we're changing this, I would change it to be properly snake case: `editor/export_as_mesh_library`.